### ISSUE TITLE
rtgdk_python3 - Conflicting Dependencies

### ISSUE THREAD

**Issue by nishantc7** [spdx/spdx-online-tools#298]
@rtgdk 
Trying to use rtgdk_python3 branch, which seems to be the one most recently updated python3 migration , it has conflicting dependencies 
```ERROR: Cannot install -r requirements.txt (line 16) and jpype1==1.2.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested jpype1==1.2.0
    spdx-license-matcher 0.0.1 depends on jpype1==1.0.1

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

**Comment by nishantc7** [comment]
Should I create a PR updating requirements.txt setting `jpype1==1.0.1` if that's okay?

**Comment by Ugtan** [comment]
@nishantc7 The jpype versions 1.0 and 1.1 has memory leak problem in strings that's why the dependency was updated to version 1.2. The conflict you got will be fixed once https://github.com/spdx/spdx-license-matcher/pull/13 this PR is merged.

**Comment by nishantc7** [comment]
> @nishantc7 The jpype versions 1.0 and 1.1 has memory leak problem in strings that's why the dependency was updated to version 1.2. The conflict you got will be fixed once https://github.com/spdx/spdx-license-matcher/pull/13 this PR is merged.

Oh, alright thanks for letting me know. Closing this now.

**Comment by rtgdk** [comment]
@nishantc7 I merged that PR spdx/spdx-license-matcher#13 and release a new version https://github.com/spdx/spdx-license-matcher/releases/tag/v2.1 and updated requirements.txt in `rtgdk_python3` branch with this release build. Now you should not face this issue while running pip install.
